stable_baselines3/common/logger.py

```diff
@@ -65,10 +71,15 @@ def __init__(self, filename_or_file: Union[str, TextIO]):
             self.file = filename_or_file
             self.own_file = False
 
-    def writekvs(self, kvs: Dict) -> None:
+    def writekvs(self, kvs: Dict, step: int) -> None:
         # Create strings for printing
         key2str = {}
         for (key, val) in sorted(kvs.items()):
+            if val[1] is not None:
+                if 'stdout' in val[1]:
+                    continue
+            val = round(val[0], 5)
```

Comment:
?

Reply:
I thought rounding would look nicer as some values can be very long

Reply:
I've removed it, not much of a difference anyways